extensions/wikia/AdEngine/AdEngine2ContextService.class.php

```diff
@@ -23,8 +25,7 @@ public function getContext( Title $title, $skinName ) {
 
 			$pageFairDetectionKey = AdEngine2Resource::getKey( 'wikia.ext.adengine.pf.detection' );
 			$pageFairDetectionUrl = ResourceLoader::makeCustomURL( $wg->Out, [ $pageFairDetectionKey ], 'scripts' );
-
-			$prebidBidderUrl = AssetsManager::getInstance()->getURL( 'prebid_prod_js', $type );
+			$prebidBidderUrl = AssetsManager::getInstance()->getURL( empty( $wgAllInOne ) ? 'prebid_debug_js' : 'prebid_prod_js' , $type );
```

Comment:
`$wgAllInOne` and generally wg globals were proven to be faster than using `$wg->`